schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -550,6 +571,130 @@
         "InstallerUrl",
         "InstallerSha256"
       ]
+    },
+    "NestedInstaller": {
+      "type": "object",
+      "properties": {
+        "InstallerLocale": {
+          "$ref": "#/definitions/Locale"
+        },
+        "Platform": {
+          "$ref": "#/definitions/Platform"
+        },
+        "MinimumOSVersion": {
+          "$ref": "#/definitions/MinimumOSVersion"
+        },
+        "Architecture": {

```

Comment:
>Architecture [](http://example.com/codeflow?start=9&length=12)

shall this one and below use the common definition?

Reply:
Created a common definition for Architecture and InstallerUrl for consistency